### ISSUE TITLE
Assets Library > Links: Can enter invalid URLs

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#3855]
### Expected behavior
Should validate the correct URL regex format

### Actual behavior
Assets Library > Links: Can enter invalid URLs

### Steps to reproduce the problem
- Login as Manager
- Click on a project > Assets Library {https://connect.topcoder-dev.com/projects/9057/assets}
- Select Links > Add New
- Enter Name=ABC; URL=www.google.asdasd.sadsad.sadasd..www.google.lk
- Click Add Link (Use https://www.w3schools.com/php/phptryit.asp?filename=tryphp_func_validate_url)
 

### Screenshot/screencast

![image](https://user-images.githubusercontent.com/42398485/78627505-367ef800-7857-11ea-9f34-95daefbb3112.png)

#### Environment
- OS: Window 10
- Browser (w/version): Chrome 80.0.3987.149 (Official Build) (64-bit)
- User role (client, copilot or manager): Manager
- Account used:  `topcoderconnect+Manager@gmail.com`

**Comment by vikasrohit** [comment]
I don't think we need this kind of validation. It neither blocks any one to use the functionality nor it causes any errors. It is upto the user what he/she wants to share. We have few standard URL validations in place, but we don't want to make it unnecessary strict which may cause some issue with valid URLs as well.
Still keeping this in backlog, if we need it some day.
